metaworld/envs/mujoco/sawyer_xyz/base.py

```diff
@@ -67,8 +67,8 @@ def reset_mocap_welds(self):
 
 class SawyerXYZEnv(SawyerMocapBase, metaclass=abc.ABCMeta):
     _HAND_SPACE = Box(
-        np.array([-0.51, .379, -.05]),
-        np.array([+0.51, 1.0, .51])
+        np.array([-0.525, .35, -.0525]),
+        np.array([+0.525, 1.025, .525])
```

Comment:
Woah why so much buffer? I thought just the y_low value was too tight

Reply:
I liked the consistency of having a uniform buffer  around the space so I expanded all of them. Because if I was able to clip in 1 axis I suspect that I could clip in another axis with the right setup